src/main/scala/org/broadinstitute/dsde/workbench/sam/util/package.scala

```diff
@@ -1,6 +1,9 @@
 package org.broadinstitute.dsde.workbench.sam
 
+import scala.concurrent.duration._
+
 package object util {
+  val API_TIMING_DURATION_BUCKET: List[FiniteDuration] = List(10 millis, 100 millis, 500 millis, 1 second, 2 seconds, 5 seconds)
```

Comment:
Is there a better spot for this? also is this the proper naming convention for a global constant?

Reply:
It will put the timing samples into each bucket, lmk if you think I should adjust.

Reply:
it's probably makes sense to adjust once the PR is merged, and after seeing some data in grafana